### PR TITLE
fix(schema): make `version` type string

### PR DESCRIPTION
**Pull request by petrjasek**
it is string in ninjs spec

### CODE REVIEW

File: content_api/items/resource.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
     'priority': {'type': 'integer'},
     'uri': {'type': 'string'},
     'usageterms': {'type': 'string'},
-    'version': {'type': 'integer', 'required': True},
+    'version': {'type': 'string', 'required': True},
```
Comment (sivakuna-aap) [inline content_api/items/resource.py]: We should add `'empty': False and 'nullable': False` as per Cerberus validation specification for string if we want to allow valid string.

**Comment by petrjasek** [comment]
@sivakuna-aap right

**Comment by plamut** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**